packages/cli/src/oclif/commands/team/remove.js

```diff
@@ -8,21 +8,31 @@ const {
 } = require('../../../utils/api');
 const { BASE_ENDPOINT } = require('../../../constants');
 
-const roleName = (role) => (role === 'collaborator' ? 'admin' : 'subscriber');
+const roleName = (role) =>
+  role === 'collaborator'
+    ? 'admin'
+    : role === 'subscriber'
+    ? 'subscriber'
+    : 'collaborator';
 
 class TeamRemoveCommand extends ZapierBaseCommand {
   async perform() {
     this.startSpinner('Loading team members');
-    const { admins, subscribers } = await listEndpointMulti(
-      { endpoint: 'collaborators', keyOverride: 'admins' },
-      {
-        endpoint: (app) =>
-          `${BASE_ENDPOINT}/api/platform/v3/integrations/${app.id}/subscribers`,
-        keyOverride: 'subscribers',
-      }
-    );
+    const { admins, subscribers, limitedCollaborators } =
+      await listEndpointMulti(
```

Comment:
seems like this is the same as the new code in `get`. can we combine them? If there's a shared function,`get` can call it and print the results. `remove` can use it to pass to `choices` below